fe/fe-core/src/main/java/org/apache/doris/datasource/property/fileformat/AvroFileFormatProperties.java

```diff
@@ -0,0 +1,56 @@
+// Licensed to the Apache Software Foundation (ASF) under one
+// or more contributor license agreements.  See the NOTICE file
+// distributed with this work for additional information
+// regarding copyright ownership.  The ASF licenses this file
+// to you under the Apache License, Version 2.0 (the
+// "License"); you may not use this file except in compliance
+// with the License.  You may obtain a copy of the License at
+//
+//   http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing,
+// software distributed under the License is distributed on an
+// "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+// KIND, either express or implied.  See the License for the
+// specific language governing permissions and limitations
+// under the License.
+
+package org.apache.doris.datasource.property.fileformat;
+
+import org.apache.doris.nereids.exceptions.AnalysisException;
+import org.apache.doris.proto.InternalService.PFetchTableSchemaRequest;
+import org.apache.doris.thrift.TFileAttributes;
+import org.apache.doris.thrift.TFileFormatType;
+import org.apache.doris.thrift.TFileTextScanRangeParams;
+import org.apache.doris.thrift.TResultFileSinkOptions;
+
+import java.util.Map;
+
+public class AvroFileFormatProperties extends FileFormatProperties {
+    public AvroFileFormatProperties(TFileFormatType fileFormatType) {
+        super(fileFormatType);
+    }
+
+    @Override
+    public void analyzeFileFormatProperties(Map<String, String> formatProperties, boolean isRemoveOriginProperty)
+            throws AnalysisException {
+    }
+
+    @Override
+    public PFetchTableSchemaRequest toPFetchTableSchemaRequest() {
```

Comment:
All the implements of `toPFetchTableSchemaRequest` and `toTResultFileSinkOptions` are return null? What is these methods for?

Reply:
I don't think it is suitable to let a "Properties" class to generate a "fetch schema request"

Reply:
> All the implements of `toPFetchTableSchemaRequest` and `toTResultFileSinkOptions` are return null? What is these methods for?

These methods are used to generate `PFetchTableSchemaRequest` and `TResultFileSinkOptions`. I will implement these methods in the next few pr, so it returns `null` currently.

Reply:
> I don't think it is suitable to let a "Properties" class to generate a "fetch schema request"

Yeah, I'll delete these abstract method.